.github/workflows/tests.yaml

```diff
@@ -75,7 +75,102 @@ jobs:
           # Make sure to add to GitHub secrets!
           token: ${{secrets.CODECOV_TOKEN}}
           file: ./directory/coverage.txt
+ 
+  gds_admin_test:
+    name: GDS Admin UI Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.8.0
+          cache: "yarn"
+          cache-dependency-path: ./web/gds-admin-ui/yarn.lock
+
+      - name: Install Dependencies
+        working-directory: ./web/gds-admin-ui
+        run: yarn 
+
+      - name: Run Tests
+        working-directory: ./web/gds-admin-ui
+        env: 
+          #maybe we can make it dynamic like ${{secrets.REACT_APP_GDS_API_ENDPOINT}}
```

Comment:
We can if you'd like - it's not exactly a secret, just a test configuration. If you feel it's hidden in the actions configuration, we could make these values the defaults in the code if you'd prefer.